docs/source/model_sharing.rst

```diff
@@ -171,15 +171,22 @@ Add a model card
 ^^^^^^^^^^^^^^^^
 
 To make sure everyone knows what your model can do, what its limitations and potential bias or ethetical
-considerations, please add a README.md model card to the 🤗 Transformers repo under `model_cards/`. It should be named
-`README.md` and follow `this template <https://github.com/huggingface/model_card>`__.
+considerations, please add a README.md model card to the 🤗 Transformers repo under `model_cards/`. It should then be
+placed in a subfolder with your username or organization, then another subfolder named like your model
+(`awesome-name-you-picked`). It should be named `README.md` and follow

```

Comment:
```suggestion
(`awesome-name-you-picked`). 
Or just click on the "Create a model card on GitHub" button on the model page, it will get you directly to the right location.
If you need one, here's a model card template (meta-suggestions are welcome):

```

Reply:
LGTM with the following tweaks

Reply:
Will reformulate (cause here's a model card template: this template is a bit weird).